loadtest/control/gencontroller/utils.go

```diff
@@ -31,3 +31,18 @@ func pickAction(actions map[string]userAction) (*userAction, error) {
 	}
 	return nil, errors.New("should not be able to reach this point")
 }
+
+// shouldMakeLongRunningThreads returns if a long thread should be created
+// TODO: The rates and logic in this function should be made configurable
+func shouldMakeLongRunningThread(channelId string) bool {
+	// 2% of the the time we check if we should make a long running thread
+	// this way we don't make all long running threads near the start
```

Comment:
This is also left over from the time this code was in the simulcontroller. I didn't want all long running threads to be created near the start. This wouldn't be an accurate representation of normal server usage. I wanted the long running threads to be created over time so that the interactions with these threads would also be spread out over time.

I suppose this doesn't matter in the gencontroller as much as it did in the simulcontroller, though I still think its better for the creation to be spread out over a period of time. Of course, the 2% check isn't very realistic, thus the comment above the function that the rate and logic in this function should be configurable. 
We also only create one long running thread per channel, which isn't very realistic either, but I think it works. I didn't want to overload the config with too many settings, but we will have to eventually.